ee/server/service/teams.go

```diff
@@ -147,7 +147,21 @@ func (svc *Service) ModifyTeamAgentOptions(ctx context.Context, teamID uint, opt
 		team.Config.AgentOptions = nil
 	}
 
-	return svc.ds.SaveTeam(ctx, team)
+	tm, err := svc.ds.SaveTeam(ctx, team)
+	if err != nil {
+		return nil, err
```

Comment:
nit:  here and below `ctxerr.Wrap`? Although I see other errors returned without wrapping in this file, so a very minor nit.

Reply:
Yeah, I think there may be a bit more context that could be added. In this case I was just keeping the previous behaviour of returning the `SaveTeam` error directly, but in the case of `NewActivity` I did think about adding a context but noticed that it was already wrapped in a ctxerr in the datastore method. But it could still benefit from additional context here (e.g. indicating which activity type it failed to created). I'll add this.